src/pages/_document.tsx

```diff
@@ -1,13 +1,32 @@
-import { Html, Head, Main, NextScript } from "next/document"
-
-export default function Document() {
-  return (
-    <Html>
-      <Head />
-      <body>
-        <Main />
-        <NextScript />
-      </body>
-    </Html>
-  )
+import { Lang } from "@/lib/types"
+import { isLangRightToLeft } from "@/lib/utils/translations"
+import NextDocument, {
+  Html,
+  Head,
+  Main,
+  NextScript,
+  DocumentContext,
+} from "next/document"
+
+class Document extends NextDocument {
+  static async getInitialProps(ctx: DocumentContext) {
+    return await NextDocument.getInitialProps(ctx)
+  }
```

Comment:
Per [response to the GH Issue](https://github.com/ethereum/ethereum-org-fork/issues/78#issuecomment-1793688702) and my follow-up, maybe this can simply be removed? I have not found issue locally not having this, but need to be allowed to check the preview deploy to see if anything goes wrong after build.

```suggestion
```